src/dev/flang/be/c/C.java

```diff
@@ -431,6 +431,7 @@ public CStmnt match(int s, AbstractInterpreter<CExpr, CStmnt> ai, CExpr sub)
           var ctags = new List<CExpr>();
```

Comment:
I think I prefer having only one variable, the original code is already messy enough.

```suggestion
          var ctagNums = new List<Integer>();
```